test-app/package.json

```diff
@@ -0,0 +1,125 @@
+    "@glimmer/tracking": "^1.1.2",
+    "@glint/environment-ember-loose": "^1.2.2",
+    "@glint/template": "^1.2.2",
+    "@tsconfig/ember": "^3.0.2",
+    "@types/ember": "^4.0.10",
+    "@types/ember-data": "^4.4.15",
+    "@types/ember-data__adapter": "^4.0.5",
+    "@types/ember-data__model": "^4.0.4",
+    "@types/ember-data__serializer": "^4.0.5",
+    "@types/ember-data__store": "^4.0.6",
+    "@types/ember__application": "^4.0.10",
+    "@types/ember__array": "^4.0.9",
+    "@types/ember__component": "^4.0.21",
+    "@types/ember__controller": "^4.0.11",
+    "@types/ember__debug": "^4.0.7",
+    "@types/ember__destroyable": "^4.0.4",
+    "@types/ember__engine": "^4.0.10",
+    "@types/ember__error": "^4.0.5",
+    "@types/ember__helper": "^4.0.5",
+    "@types/ember__modifier": "^4.0.8",
+    "@types/ember__object": "^4.0.11",
+    "@types/ember__owner": "^4.0.8",
+    "@types/ember__polyfills": "^4.0.5",
+    "@types/ember__routing": "^4.0.19",
+    "@types/ember__runloop": "^4.0.8",
+    "@types/ember__service": "^4.0.8",
+    "@types/ember__string": "^3.16.3",
+    "@types/ember__template": "^4.0.5",
+    "@types/ember__test": "^4.0.5",
+    "@types/ember__utils": "^4.0.6",
+    "@types/qunit": "^2.19.9",
+    "@types/rsvp": "^4.0.8",
+    "@typescript-eslint/eslint-plugin": "^6.14.0",
+    "@typescript-eslint/parser": "^6.15.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "concurrently": "^8.2.2",
+    "ember-auto-import": "^2.7.0",
+    "ember-cli": "~5.5.0",
+    "ember-cli-app-version": "^6.0.1",
+    "ember-cli-babel": "^8.2.0",
+    "ember-cli-clean-css": "^3.0.0",
+    "ember-cli-dependency-checker": "^3.3.2",
+    "ember-cli-htmlbars": "^6.3.0",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-terser": "^4.0.2",
+    "ember-data": "~5.3.0",
+    "ember-fetch": "^8.1.2",
+    "ember-load-initializers": "^2.1.2",
+    "ember-modifier": "^4.1.0",
+    "ember-page-title": "^8.1.0",
+    "ember-qunit": "^8.0.2",
+    "ember-resolver": "^11.0.1",
+    "ember-source": "~5.5.0",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-template-lint": "^5.13.0",
+    "ember-try": "^3.0.0",
+    "eslint": "^8.55.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^11.11.1",
+    "eslint-plugin-n": "^16.4.0",
+    "eslint-plugin-prettier": "^5.0.1",
+    "eslint-plugin-qunit": "^8.0.1",
+    "loader.js": "^4.7.0",
+    "prettier": "^3.1.1",
+    "qunit": "^2.20.0",
+    "qunit-dom": "^2.0.0",
+    "stylelint": "^15.11.0",
+    "stylelint-config-standard": "^34.0.0",
+    "stylelint-prettier": "^4.1.0",
+    "tracked-built-ins": "^3.3.0",
+    "typescript": "^5.3.3",
+    "webpack": "^5.89.0",
+    "@glint/core": "^1.2.2",
+    "@glint/environment-ember-template-imports": "^1.2.1",
+    "ember-eslint-parser": "^0.2.5",
+    "ember-template-imports": "^4.0.0",
+    "ember-welcome-page": "^7.0.2",
+    "prettier-plugin-ember-template-tag": "^2.0.0",
+    "rimraf": "^5.0.1"
```

Comment:
The dependencies aren't ordered alphabetically. That will cause unnecessary churn when PNPM or RenovateBot reorders them. Could you please fix?